client-app/core/api/graphql/config/error-handler.ts

```diff
@@ -13,10 +13,17 @@ export const errorHandlerLink = onError(({ networkError, graphQLErrors }) => {
   const passwordExpired = hasErrorCode(graphQLErrors, GraphQLErrorCode.PasswordExpired);
 
   if (userLockedError) {
-    broadcast.emit(userLockedEvent, undefined, TabsType.ALL);
+    void broadcast.emit(userLockedEvent, undefined, TabsType.ALL);
   }
 
   if (passwordExpired) {
-    broadcast.emit(passwordExpiredEvent, undefined, TabsType.CURRENT);
+    void broadcast.emit(passwordExpiredEvent, undefined, TabsType.CURRENT);
+  }
+
+  if (graphQLErrors) {
+    const errors = graphQLErrors.map((error) => `[GraphQL error]: Message: ${error.message}`);
+    errors.forEach((error) => {
+      throw new Error(error);
```

Comment:
why don't throw a `new GraphQLError` ? it can contains some useful context
if you want you can still modify the error message like that:
```ts
errors.forEach((error) => {
  throw new Error(`[GraphQL error]: Message: ${error.message}`, {
    originalError: error,
  });
}
```